docs/tutorial-create.rst

```diff
@@ -0,0 +1,180 @@
+data. Many of these pipelines are available via Resolwe SDK, and are listed in
+the `Process catalog`_ of the `Resolwe Bioinformatics documentation`_.
+
+.. _Process catalog: http://resolwe-bio.readthedocs.io/en/latest/catalog.html
+.. _Resolwe Bioinformatics documentation: http://resolwe-bio.readthedocs.io
+
+Next step to make is to align reads to a genome. We will use HISAT2 aligner, which is wrapped in a process with slug ``alignment-hisat2``. INputs and outputs of this process are described in `HISAT2 process catalog`_. We will define input files and process runs its
+algorithm that transforms inputs into outputs.
+
+.. _HISAT2 process catalog: https://resolwe-bio.readthedocs.io/en/latest/catalog-definitions.html#process-alignment-hisat2
+
+.. literalinclude:: files/tutorial-create.py
+   :lines: 66-75
+
+Lets take a closer look to the code above. We defined the alignment process, by its slug ``'alignment-hisat2'``.
+For inputs we defined data objects ``reads`` and ``genome``. ``Reads`` object was created with 'upload-fastq-single'
+or 'upload-fastq-paired' processor, while ``genome`` data object was already on the platform. We just used it's slug to identify it.
+The ``alignment-hisat2`` processor will automatically took the right files from data objects, specified in inputs and
+create output files: ``bam`` alignment file, ``bai`` index and somo more...
+
+You probably noticed that we get the result almost instantly, while the
+typical assembling process runs for hours. This is because
+processing runs asynchronously, so the returned data object does not
+have an OK status or outputs when returned.
+
+.. literalinclude:: files/tutorial-create.py
+   :lines: 77-84
+
+Status ``OK`` indicates that processing has finished successfuly, but you will also find other statuses. They are given
+with two-letter abbreviations. To understand their meanig, check the
+:obj:`status reference <resdk.resources.Data.status>`. When processing is done, all outputs are witten to disk and you
+can inspect them:
+
+.. literalinclude:: files/tutorial-create.py
+   :lines: 86-87
+
+Unitl now, we used ``run()`` method twice: to upload reads (yes, uploading files is just a matter of using an upload
+process) and to run alignment. You can check the full signature of the :obj:`run() <resdk.Resolwe.run>` method.
+
+Run workflows
+=============
+
+As typical analysis of data is performed by running a process that outputs some data. This data is fed as input into
+another proces that produces another set of outputs. This output is then again fed into another proces... Basically
+analysis is a sequence of processes. Sometimes, this sequence is so commonly used that one want to simplify it's
+execution. This can be done by using so called "workflow". Workflows are special processes: then run a stack of
+processes. On the outside, they look exactly the same as a normal process: They have a process slug, inputs, outputs...
```

Comment:
On the outside, they look exactly the same as a normal process and have a process slug, inputs, outputs...